transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringServerSocketChannelConfig.java

```diff
@@ -21,5 +21,12 @@
 
     IOUringServerSocketChannelConfig(AbstractIOUringServerChannel channel) {
         super(channel);
+        setReuseAddress(true);
```

Comment:
I'm not able to recreate the problem on my end. Are you sure the first process had really been terminated?
`SO_REUSEADDR` sounds like something that should be explicitly opted into by the integrator code.

Reply:
that's strange.. I'm sure the first process have been terminated, I think the reason the server file descriptor is not closed after the termination https://docs.fedoraproject.org/en-US/Fedora_Security_Team/1/html/Defensive_Coding/sect-Defensive_Coding-Tasks-Descriptors-Child_Processes.html
on epoll that we do not close server file descriptor when sending SIGTERM
I just saw that set SO_REUSEADDR here https://github.com/netty/netty/blob/4.1/transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollServerSocketChannelConfig.java#L39

Reply:
are you sure that's right HEAD commit? https://github.com/1Jo1/netty/commit/97b4537ab1e3bbb62f59802b18808fa48b38ad27  I just testet it again and I'm able to do it
Log Message:
`Exception in thread "main" io.netty.channel.unix.Errors$NativeIoException: bind(..) failed: Address already in use`
` I get EventLoop Accept Res: -9`

Reply:
I tested 8c9b874a2d40a894e9ab7a57eb85e9d3a5027c52 (first running prototype). When I comment out the reuse port on the HEAD commit, then I also get `Address already in use`. It's not really obvious which change introduced the failure, in between those two commits.

Reply:
ah okay the error occurred when I used socket non blocking as far as I know https://github.com/netty/netty/commit/97b4537ab1e3bbb62f59802b18808fa48b38ad27
https://github.com/netty/netty/blob/97b4537ab1e3bbb62f59802b18808fa48b38ad27/transport-native-io_uring/src/main/java/io/netty/channel/uring/IOUringSubmissionQueue.java#L105

Reply:
interesting, i got the same bug in epolll when i removed that
https://github.com/netty/netty/blob/4.1/transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollServerSocketChannelConfig.java#L39

Reply:
@1Jo1 Okay, in that case I think it's fine to just follow the precedent set by epoll.